runtime/redirector/redirector.c

```diff
@@ -706,9 +706,11 @@ chooseJVM(JavaVMInitArgs *args, char *retBuffer, size_t bufferLength)
 
 		/* direct user to OpenJ9 build configurations to properly generate the requested build. */
 		if (DEFAULT_DIR == basePointer) {
+			fprintf(stdout, "Start the VM without the -Xnocompressedrefs option or with -Xcompressedrefs.\n");
```

Comment:
This is a great idea.  Two thoughts: can we use a single fprintf and a more user friendly message?
Maybe something along the lines of:

```This JVM package only includes the '-Xcompressedrefs' configuration.  Please run the VM without specifying the '-Xnocompressedrefs' option or by specifying the '-Xcompressedrefs' option.\n  To compile the other configuration, please run configure with '--with-noncompressedrefs```

Reply:
that sounds much better. I've updated the message.